### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by pawanpinjarkar**
Update the readme to point to sample manifests to try out the deployment
Ref: https://issues.redhat.com/browse/AGENT-67

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,18 +13,19 @@ These are all built into the binary at compilation time.
...
-* manifests/pull-secret.yaml
-* manifests/cluster-deployment.yaml
-* manifests/agent-cluster-install.yaml
-* manifests/infraenv.yaml
+* [manifests/pull-secret.yaml](https://github.com/openshift-metal3/dev-scripts/blob/master/agent/03_agent_deployment.sh#L142-L152)
+   - Your pull secret JSON data can be [obtained from the OpenShift Console](https://console.redhat.com/openshift/install/pull-secret).
+* [manifests/cluster-deployment.yaml](https://github.com/openshift-metal3/dev-scripts/blob/master/agent/03_agent_deployment.sh#L97-L120)
+* [manifests/agent-cluster-install.yaml](https://github.com/openshift-metal3/dev-scripts/blob/master/agent/03_agent_deployment.sh#L73-L95)
+* [manifests/infraenv.yaml](https://github.com/openshift-metal3/dev-scripts/blob/master/agent/03_agent_deployment.sh#L122-L138)
+* [manifests/nmstateconfig.yaml](https://github.com/openshift-metal3/dev-scripts/blob/master/agent/03_agent_deployment.sh#L159-L194)
```
Comment (bfournie) [inline README.md]: Maybe I'm missing something but are we going to have to keep the line numbers updated when dev-scripts changes?

Comment (pawanpinjarkar) [reply README.md]: I originally preferred adding a `samples` directory under `manifests` directory and updating the ReadMe with that link but there was a discussion to just point it to dev-scripts. I am in favor of adding a samples dir in this repo. WDYT? Any other options? @bfournie Thank you.

Comment (pawanpinjarkar) [reply README.md]: Added samples under `./manifests/samples` dir

**Pull request merged**